work/v1/types.go

```diff
@@ -39,6 +39,14 @@ type ManifestWorkSpec struct {
 	// ManifestConfigs represents the configurations of manifests defined in workload field.
 	// +optional
 	ManifestConfigs []ManifestConfigOption `json:"manifestConfigs,omitempty"`
+
+	// Executor is the configuration for customizing how work agent executes/applies the workloads to the
```

Comment:
Let's get more specific than customizing.  Setting the executor tells the work agent to check whether the executor has permissions to perform mutations on the individual resources in manifestwork.

Reply:
clarified the godoc